src/txdb.cpp

```diff
@@ -46,6 +46,9 @@ static const char DB_MMR_LENGTH = 'M';
 static const char DB_MMR_NODE = 'm';
 static const char DB_MMR_ROOT = 'r';
 
+static const char DB_SUBTREE_LATEST = 'e';
+static const char DB_SUBTREE_DATA = 'n';
+
```

Comment:
I checked that all of these types are still unique, and @str4d checked that types in Bitcore Core that collide are unlikely to need to be backported.